gdc_client/utils.py

```diff
@@ -0,0 +1,14 @@
+from urllib import urlencode
```

Comment:
The `requests` library actually supports this out of the box.

```python
>>> payload = {'key1': 'value1', 'key2': 'value2'}
>>> r = requests.get('http://httpbin.org/get', params=payload)
>>> print(r.url)
http://httpbin.org/get?key2=value2&key1=value1
```

http://docs.python-requests.org/en/master/user/quickstart/#passing-parameters-in-urls

Reply:
right, the idea is that it won't support stuff like `url?flag1&flag2` and that's how `latest` and some other flags (`tar`?) are passed

Reply:
```python
In [1]: import requests

In [2]: requests.post('https://api.gdc.cancer.gov/status', params='a&b&c').url
Out[2]: u'https://api.gdc.cancer.gov/status?a&b&c'
```
The `params` keyword arg accepts any string or dictionary.